stdlib/public/core/VarArgs.swift

```diff
@@ -168,6 +168,12 @@ extension Int : CVarArg {
   }
 }
 
+extension Bool : CVarArg {
+  public var _cVarArgEncoding: [Int] {
+    return _encodeBitsAsWords(self)
```

Comment:
This should promote to Int32 here, like Int16 and Int8 do, to ensure correct encoding on big-endian systems.